[16:28:37 18-12-2017] (Debug) main.py::<module>.1036: Server Reload...
[16:29:05 18-12-2017] (Debug) main.py::<module>.1037: Server Reload...
[16:29:18 18-12-2017] (Debug) main.py::<module>.1037: Server Reload...
[16:29:21 18-12-2017] (Debug) main.py::<module>.1037: Server Reload...
[16:31:26 18-12-2017] (Debug) main.py::<module>.1040: Server Reload...
[16:31:27 18-12-2017] (Debug) main.py::<module>.1040: Server Reload...
[16:31:31 18-12-2017] (Success) main.py::index.94: Lade 2 Posts
[16:31:35 18-12-2017] (Success) main.py::index.94: Lade 2 Posts
[16:31:38 18-12-2017] (Success) main.py::index.94: Lade 2 Posts
[16:31:54 18-12-2017] (Debug) main.py::<module>.1039: Server Reload...
[16:32:09 18-12-2017] (Error) main.py::login.129: Gescheiterter Login (Datenbankfehler): awd@awd.de
[16:32:21 18-12-2017] (Error) main.py::login.140: Gescheiterter Login (Übereinstimmung): marc-friedrich94@web.de
[16:32:41 18-12-2017] (Success) main.py::login.162: Erfolgreicher Login: marc-friedrich94@web.de (ADMINISTRATOR)
[16:38:20 18-12-2017] (Debug) main.py::<module>.1040: Server Reload...
[16:38:33 18-12-2017] (Success) main.py::login.162: Erfolgreicher Login: marc-friedrich94@web.de (ADMINISTRATOR)
[16:38:40 18-12-2017] (Success) main.py::logout.185: Benutzer wurde ausgeloggt: marc-friedrich94@web.de
[16:39:18 18-12-2017] (Debug) main.py::<module>.1041: Server Reload...
[16:39:53 18-12-2017] (Debug) main.py::<module>.1042: Server Reload...
[16:39:55 18-12-2017] (Debug) main.py::<module>.1041: Server Reload...
[16:40:06 18-12-2017] (Debug) main.py::<module>.1040: Server Reload...
[16:40:09 18-12-2017] (Debug) main.py::<module>.1039: Server Reload...
[16:40:13 18-12-2017] (Debug) main.py::<module>.1038: Server Reload...
[16:40:45 18-12-2017] (Debug) main.py::<module>.1039: Server Reload...
[16:40:47 18-12-2017] (Debug) main.py::<module>.1038: Server Reload...
[16:41:15 18-12-2017] (Debug) main.py::<module>.1039: Server Reload...
[16:41:32 18-12-2017] (Debug) main.py::<module>.1039: Server Reload...
[16:41:42 18-12-2017] (Debug) main.py::<module>.1039: Server Reload...
[16:41:44 18-12-2017] (Debug) main.py::<module>.1039: Server Reload...
[16:42:21 18-12-2017] (Debug) main.py::<module>.1038: Server Reload...
[16:42:45 18-12-2017] (Debug) main.py::<module>.1039: Server Reload...
[16:42:58 18-12-2017] (Debug) main.py::<module>.1039: Server Reload...
[16:44:30 18-12-2017] (Debug) main.py::<module>.1040: Server Reload...
[16:44:59 18-12-2017] (Debug) main.py::<module>.1040: Server Reload...
[16:45:05 18-12-2017] (Debug) main.py::<module>.1042: Server Reload...
[16:45:07 18-12-2017] (Debug) main.py::<module>.1043: Server Reload...
[16:45:22 18-12-2017] (Debug) main.py::<module>.1044: Server Reload...
[16:46:02 18-12-2017] (Debug) main.py::<module>.1045: Server Reload...
[16:47:52 18-12-2017] (Debug) main.py::<module>.1046: Server Reload...
[16:48:00 18-12-2017] (Debug) main.py::<module>.1046: Server Reload...
[16:48:03 18-12-2017] (Debug) main.py::<module>.1044: Server Reload...
[16:48:27 18-12-2017] (Debug) main.py::<module>.1045: Server Reload...
[16:48:37 18-12-2017] (Debug) main.py::<module>.1044: Server Reload...
[16:48:59 18-12-2017] (Debug) main.py::<module>.1041: Server Reload...
[16:49:38 18-12-2017] (Debug) main.py::<module>.1042: Server Reload...
[16:49:54 18-12-2017] (Debug) main.py::<module>.1041: Server Reload...
[16:50:07 18-12-2017] (Debug) main.py::<module>.1042: Server Reload...
[16:50:12 18-12-2017] (Debug) main.py::<module>.1041: Server Reload...
[16:51:09 18-12-2017] (Debug) main.py::<module>.1049: Server Reload...
[16:51:15 18-12-2017] (Debug) main.py::<module>.1057: Server Reload...
[16:51:22 18-12-2017] (Debug) main.py::<module>.1065: Server Reload...
[16:51:29 18-12-2017] (Debug) main.py::<module>.1073: Server Reload...
[16:52:14 18-12-2017] (Debug) main.py::<module>.1081: Server Reload...
[16:53:26 18-12-2017] (Debug) main.py::<module>.1082: Server Reload...
[16:53:46 18-12-2017] (Debug) main.py::<module>.1082: Server Reload...
[16:53:58 18-12-2017] (Debug) main.py::<module>.1083: Server Reload...
[16:54:18 18-12-2017] (Debug) main.py::<module>.1092: Server Reload...
[16:54:38 18-12-2017] (Debug) main.py::<module>.1093: Server Reload...
[16:54:44 18-12-2017] (Debug) main.py::<module>.1094: Server Reload...
[16:54:53 18-12-2017] (Debug) main.py::<module>.1095: Server Reload...
[16:55:03 18-12-2017] (Debug) main.py::<module>.1096: Server Reload...
[16:55:10 18-12-2017] (Debug) main.py::<module>.1097: Server Reload...
[16:56:39 18-12-2017] (Debug) main.py::<module>.1098: Server Reload...
[16:57:49 18-12-2017] (Debug) main.py::<module>.1099: Server Reload...
[16:57:51 18-12-2017] (Debug) main.py::<module>.1101: Server Reload...
[16:58:16 18-12-2017] (Debug) main.py::<module>.1102: Server Reload...
[16:58:46 18-12-2017] (Success) main.py::login.162: Erfolgreicher Login: marc-friedrich94@web.de (ADMINISTRATOR)
[16:59:07 18-12-2017] (Success) main.py::post_message.368: Neue Nachricht von Benuzter marc-friedrich94@web.de geposted
[16:59:53 18-12-2017] (Debug) main.py::<module>.1101: Server Reload...
[17:00:08 18-12-2017] (Debug) main.py::<module>.1100: Server Reload...
[17:00:23 18-12-2017] (Debug) main.py::<module>.1100: Server Reload...
[17:00:53 18-12-2017] (Debug) main.py::<module>.1101: Server Reload...
[17:00:57 18-12-2017] (Debug) main.py::<module>.1101: Server Reload...
[17:01:25 18-12-2017] (Debug) main.py::<module>.1102: Server Reload...
[17:01:48 18-12-2017] (Success) main.py::vote.407: Benutzer marc-friedrich94@web.de hat bereits für Post 5 abgestimmt
[17:02:43 18-12-2017] (Debug) main.py::<module>.1103: Server Reload...
[17:03:23 18-12-2017] (Debug) main.py::<module>.1104: Server Reload...
[17:05:42 18-12-2017] (Debug) main.py::<module>.1105: Server Reload...
[17:06:26 18-12-2017] (Debug) main.py::<module>.1106: Server Reload...
[17:07:07 18-12-2017] (Debug) main.py::<module>.1107: Server Reload...
[17:07:50 18-12-2017] (Debug) main.py::<module>.1109: Server Reload...
[17:10:37 18-12-2017] (Debug) main.py::<module>.1110: Server Reload...
[17:10:46 18-12-2017] (Debug) main.py::<module>.1111: Server Reload...
[17:11:25 18-12-2017] (Debug) main.py::<module>.1112: Server Reload...
[17:11:38 18-12-2017] (Debug) main.py::<module>.1113: Server Reload...
[17:46:56 18-12-2017] (Debug) main.py::<module>.1114: Server Reload...
[17:47:15 18-12-2017] (Debug) main.py::<module>.1115: Server Reload...
[17:47:42 18-12-2017] (Debug) main.py::<module>.1116: Server Reload...
[17:50:56 18-12-2017] (Debug) main.py::<module>.1116: Server Reload...
[17:51:34 18-12-2017] (Debug) main.py::<module>.1117: Server Reload...
[17:51:36 18-12-2017] (Debug) main.py::<module>.1117: Server Reload...
[17:52:51 18-12-2017] (Debug) main.py::<module>.1118: Server Reload...
[17:53:00 18-12-2017] (Debug) main.py::<module>.1116: Server Reload...
[17:53:38 18-12-2017] (Debug) main.py::<module>.1117: Server Reload...
[17:54:02 18-12-2017] (Debug) main.py::<module>.1118: Server Reload...
[17:54:37 18-12-2017] (Debug) main.py::<module>.1119: Server Reload...
[17:55:08 18-12-2017] (Debug) main.py::<module>.1120: Server Reload...
[17:56:19 18-12-2017] (Debug) main.py::<module>.1121: Server Reload...
[17:56:53 18-12-2017] (Debug) main.py::<module>.1123: Server Reload...
[17:57:13 18-12-2017] (Debug) main.py::<module>.1124: Server Reload...
[17:57:34 18-12-2017] (Debug) main.py::<module>.1126: Server Reload...
[17:58:10 18-12-2017] (Debug) main.py::<module>.1127: Server Reload...
[17:58:15 18-12-2017] (Debug) main.py::<module>.1128: Server Reload...
[17:58:28 18-12-2017] (Debug) main.py::<module>.1127: Server Reload...
[17:58:40 18-12-2017] (Debug) main.py::<module>.1128: Server Reload...
[17:58:45 18-12-2017] (Debug) main.py::<module>.1128: Server Reload...
[17:59:04 18-12-2017] (Debug) main.py::<module>.1130: Server Reload...
[17:59:13 18-12-2017] (Debug) main.py::<module>.1131: Server Reload...
[18:03:08 18-12-2017] (Debug) main.py::<module>.1132: Server Reload...
[18:04:15 18-12-2017] (Debug) main.py::<module>.1133: Server Reload...
[18:04:29 18-12-2017] (Debug) main.py::<module>.1134: Server Reload...
[18:04:31 18-12-2017] (Debug) main.py::<module>.1134: Server Reload...
[18:05:02 18-12-2017] (Debug) main.py::<module>.1135: Server Reload...
[18:05:23 18-12-2017] (Debug) main.py::<module>.1136: Server Reload...
[18:05:59 18-12-2017] (Debug) main.py::<module>.1137: Server Reload...
[18:06:07 18-12-2017] (Debug) main.py::<module>.1137: Server Reload...
[18:06:28 18-12-2017] (Debug) main.py::<module>.1138: Server Reload...
[18:06:38 18-12-2017] (Debug) main.py::<module>.1137: Server Reload...
[18:07:05 18-12-2017] (Debug) main.py::<module>.1138: Server Reload...
[18:07:09 18-12-2017] (Debug) main.py::<module>.1138: Server Reload...
[18:13:58 18-12-2017] (Debug) main.py::<module>.1139: Server Reload...
[18:14:19 18-12-2017] (Debug) main.py::<module>.1141: Server Reload...
[18:14:56 18-12-2017] (Debug) main.py::<module>.1142: Server Reload...
[18:15:12 18-12-2017] (Debug) main.py::<module>.1143: Server Reload...
[18:15:44 18-12-2017] (Debug) main.py::<module>.1143: Server Reload...
[18:16:12 18-12-2017] (Debug) main.py::<module>.1145: Server Reload...
[18:16:14 18-12-2017] (Debug) main.py::<module>.1145: Server Reload...
[18:16:18 18-12-2017] (Debug) main.py::<module>.1145: Server Reload...
[18:18:28 18-12-2017] (Debug) main.py::<module>.1146: Server Reload...
[18:18:29 18-12-2017] (Debug) main.py::<module>.1146: Server Reload...
[18:19:59 18-12-2017] (Debug) main.py::<module>.1149: Server Reload...
[18:20:41 18-12-2017] (Debug) main.py::<module>.1151: Server Reload...
[18:21:45 18-12-2017] (Debug) main.py::<module>.1152: Server Reload...
[18:22:18 18-12-2017] (Debug) main.py::<module>.1153: Server Reload...
[18:22:48 18-12-2017] (Debug) main.py::<module>.1154: Server Reload...
[18:23:29 18-12-2017] (Debug) main.py::<module>.1156: Server Reload...
[18:24:08 18-12-2017] (Debug) main.py::<module>.1157: Server Reload...
[18:25:01 18-12-2017] (Debug) main.py::<module>.1159: Server Reload...
[18:25:40 18-12-2017] (Debug) main.py::<module>.1160: Server Reload...
[18:26:25 18-12-2017] (Debug) main.py::<module>.1163: Server Reload...
[18:26:47 18-12-2017] (Debug) main.py::<module>.1158: Server Reload...
[18:26:54 18-12-2017] (Debug) main.py::<module>.1156: Server Reload...
[18:27:30 18-12-2017] (Debug) main.py::<module>.1154: Server Reload...
[18:27:40 18-12-2017] (Debug) main.py::<module>.1150: Server Reload...
[18:28:22 18-12-2017] (Success) main.py::logout.183: Benutzer wurde ausgeloggt: marc-friedrich94@web.de
[18:29:36 18-12-2017] (Success) main.py::handle_password_reset.942: Anfrage für Passwort Reset wurde registriert. Sende Reset E-Mail an marc-friedrich94@web.de
[18:30:18 18-12-2017] (Error) main.py::confirm_password_reset.980: Übermitteltes Token stimmt nicht mit registriertem Token überein. Mögliche Attacke (SQLi) erkannt.
[18:30:34 18-12-2017] (Success) main.py::set_new_password.1029: Passwort für Benutzer 9 wurde geändert.
[18:30:35 18-12-2017] (Debug) main.py::set_new_password.1033: Logge Benutzer 9 aus, um sich mit den geänderten Credentials erneut auszuloggen.
[18:30:53 18-12-2017] (Success) main.py::handle_password_reset.942: Anfrage für Passwort Reset wurde registriert. Sende Reset E-Mail an marc-friedrich94@web.de
[18:31:19 18-12-2017] (Success) main.py::set_new_password.1029: Passwort für Benutzer 9 wurde geändert.
[18:31:20 18-12-2017] (Debug) main.py::set_new_password.1033: Logge Benutzer 9 aus, um sich mit den geänderten Credentials erneut auszuloggen.
[18:31:29 18-12-2017] (Error) main.py::login.139: Gescheiterter Login (Übereinstimmung): marc-friedrich94@web.de
[18:31:39 18-12-2017] (Success) main.py::login.160: Erfolgreicher Login: marc-friedrich94@web.de (ADMINISTRATOR)
[18:32:35 18-12-2017] (Success) main.py::change_password_handler.714: Bestätigungsmail für Passwortänderung wurde an marc-friedrich94@web.de gesendet.
[18:33:03 18-12-2017] (Success) main.py::confirm_password_change.767: Benutzer 9 hat Passwort geändert. Räume Tabelle mit alten Einträgen dieses Users auf...
[18:33:03 18-12-2017] (Debug) main.py::confirm_password_change.770: Logge User aus, um sich mit den neuen Credentials anzumelden.
[18:33:24 18-12-2017] (Success) main.py::login.160: Erfolgreicher Login: marc-friedrich94@web.de (ADMINISTRATOR)
[18:33:54 18-12-2017] (Success) main.py::change_email_handler.580: Bestätigungsmail für E-Mail Änderung wurde an marc-friedrich94@web.de gesendet.
[18:36:13 18-12-2017] (Success) main.py::change_email_handler.580: Bestätigungsmail für E-Mail Änderung wurde an marc-friedrich94@web.de gesendet.
[18:36:53 18-12-2017] (Success) main.py::confirm_email_change.632: Benutzer 9 hat seine E-Mail geändert. Räume Tabelle mit alten Einträgen dieses Users auf...
[18:36:53 18-12-2017] (Debug) main.py::confirm_email_change.635: Logge User aus, um sich mit den neuen Credentials anzumelden.
[18:37:04 18-12-2017] (Error) main.py::login.129: Gescheiterter Login (Datenbankfehler): marc-friedrich94@web.de
[18:37:13 18-12-2017] (Success) main.py::login.160: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[18:37:20 18-12-2017] (Success) main.py::admin_dashboard.280: Administrator 'marc.friedrich94@web.de hat Admin Dashboard geladen
[18:37:28 18-12-2017] (Success) main.py::admin_dashboard.280: Administrator 'marc.friedrich94@web.de hat Admin Dashboard geladen
[18:37:31 18-12-2017] (Success) main.py::admin_confirm.891: Adminstratorfunktion wurde erfolgreich bestätigt. (Methode:delete, Objekt: 4
[18:37:35 18-12-2017] (Success) main.py::logout.183: Benutzer wurde ausgeloggt: marc.friedrich94@web.de
[18:38:03 18-12-2017] (Success) main.py::login.160: 
[18:38:14 18-12-2017] (Error) main.py::admin_dashboard.265: Unbefugter Benutzer 'Ingo.piller@hans-maier.de versucht auf das Admin Dashboard zuzugreifen. Verweigere Zugriff.
[18:38:38 18-12-2017] (Error) main.py::delete_user.802: Unbefugter Benutzer Ingo.piller@hans-maier.de wollte User löschen. Mögliche Attacke erkannt.
[18:39:00 18-12-2017] (Success) main.py::post_message.366: Neue Nachricht von Benuzter Ingo.piller@hans-maier.de geposted
[18:39:11 18-12-2017] (Success) main.py::vote.424: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[18:39:18 18-12-2017] (Success) main.py::finish_vote.488: Upvote für 3 von Ingo.piller@hans-maier.de wurde registriert.
[18:39:30 18-12-2017] (Success) main.py::vote.407: Benutzer Ingo.piller@hans-maier.de hat bereits für Post 3 abgestimmt
[18:39:41 18-12-2017] (Success) main.py::vote.424: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[18:39:44 18-12-2017] (Error) main.py::finish_vote.455: Vote konnte nicht abgeschlossen werden (keine Übereinstimmung der Token)
[18:39:55 18-12-2017] (Success) main.py::vote.424: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[18:40:02 18-12-2017] (Success) main.py::finish_vote.488: Upvote für 4 von Ingo.piller@hans-maier.de wurde registriert.
[18:40:15 18-12-2017] (Success) main.py::vote.424: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[18:40:26 18-12-2017] (Success) main.py::finish_vote.488: Upvote für 6 von Ingo.piller@hans-maier.de wurde registriert.
[18:40:31 18-12-2017] (Success) main.py::logout.183: Benutzer wurde ausgeloggt: Ingo.piller@hans-maier.de
[18:40:41 18-12-2017] (Success) main.py::login.160: Erfolgreicher Login: marc.friedrich94@web.de (ADMINISTRATOR)
[18:40:48 18-12-2017] (Success) main.py::vote.424: Gebe Seite mit CSRF Token zurück. Warte auf Eingabe von Benutzer...
[18:40:55 18-12-2017] (Success) main.py::finish_vote.488: Upvote für 6 von marc.friedrich94@web.de wurde registriert.
[18:41:07 18-12-2017] (Success) main.py::logout.183: Benutzer wurde ausgeloggt: marc.friedrich94@web.de
[18:41:14 18-12-2017] (Success) main.py::login.160: 
[18:41:22 18-12-2017] (Success) main.py::vote.407: Benutzer Ingo.piller@hans-maier.de hat bereits für Post 6 abgestimmt
[18:43:33 18-12-2017] (Debug) main.py::<module>.1150: Server Reload...
